src/ExpenseTrackerGrupo2/DataAccess/Repositories/Interfaces/IBudgetRepository.cs

```diff
@@ -0,0 +1,9 @@
+using ExpenseTrackerGrupo2.DataAccess.Entities;
+
+namespace ExpenseTrackerGrupo2.DataAccess.Concretes;
+
+    public interface IBudgetRepository : IBaseRepository<Budget>
```

Comment:
that does not have space, you have to be at the same height as the namespace

Reply:
Done